<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(39, 39, 39);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="list_commitments">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>361</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(65, 65, 65);
color: rgb(255, 255, 255);
selection-color: rgb(0, 0, 0);
selection-background-color:rgb(138, 138, 138);  
alternate-background-color:  rgb(106, 106, 106);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);background-color: rgb(144, 144, 144);</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_agents">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Agents Available</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="num_agents">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{
	background-color: rgb(68, 68, 68);
	color: rgb(0, 0, 0)
}
</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="num_queue">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{
	background-color: rgb(68, 68, 68);
	color: rgb(0, 0, 0)
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="queue_title">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Queue</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_delete_alert">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_update">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Last Update:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_update_time">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="list_agents">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(65, 65, 65);
color: rgb(255, 255, 255);
selection-color: rgb(0, 0, 0);
selection-background-color:rgb(138, 138, 138);  
alternate-background-color:  rgb(106, 106, 106);</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_roi">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(65, 65, 65);
color: rgb(255, 255, 255);
selection-color: rgb(0, 0, 0);
selection-background-color:rgb(138, 138, 138);  
alternate-background-color:  rgb(106, 106, 106);</string>
    </property>
   </widget>
   <widget class="QLabel" name="queue_title_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(0, 0, 0, 0));</string>
    </property>
    <property name="text">
     <string>Queue</string>
    </property>
   </widget>
   <widget class="QLabel" name="queue_title_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(0, 0, 0, 0));</string>
    </property>
    <property name="text">
     <string>ROI</string>
    </property>
   </widget>
   <widget class="QLabel" name="queue_title_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(0, 0, 0, 0));</string>
    </property>
    <property name="text">
     <string>Support</string>
    </property>
   </widget>
   <zorder>list_commitments</zorder>
   <zorder>title_agents</zorder>
   <zorder>queue_title</zorder>
   <zorder>label_delete_alert</zorder>
   <zorder>num_queue</zorder>
   <zorder>num_agents</zorder>
   <zorder>btn_delete</zorder>
   <zorder>label_update</zorder>
   <zorder>label_update_time</zorder>
   <zorder>list_agents</zorder>
   <zorder>list_roi</zorder>
   <zorder>queue_title_2</zorder>
   <zorder>queue_title_3</zorder>
   <zorder>queue_title_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
